slnviz.py

```diff
@@ -87,7 +171,7 @@ def get_project_ids(self, nodes):
     def get_declared_project_dependency_ids(self):
         xml_proj = self.get_full_project_file_path()
         if not os.path.isfile(xml_proj):
-            print("--Project {0}-- Couldn't open project-file '{1}'".format(self.name, xml_proj))
+            log_info("--Project {0}-- Couldn't open project-file '{1}'".format(self.name, xml_proj))

```

Comment:
I would probably define this as a warning, not as info.

Reply:
@josteink I agree with you. All original prints which did not have a clear ERROR or WARNING mention were simply (too simple) changed into `log_info` . I will adapt and issue a new pull request with relevant updates.

Reply:
No need fir a new PR. Just add changes to your current branch and push the extra commits!

Reply:
Done